### PR TITLE
Applicant dashboard

### PR DESCRIPTION
**Pull request by OlenaKashuba**
- Add ApplicantDashboard component
- Add DashboardStep component
- Add StatusMessage component
- create stepsArray variable for export

* no styles added

### CODE REVIEW

File: frontend/src/stepsArray.js [hunk 1]
```diff
@@ -0,0 +1,8 @@
+export default [
+	{step: 0, details: 'Website Application'},
+	{step: 1, details: 'Khan Academy: Basics Tutorial /  Solo Learn: Basics Tutorial'},
+	{step: 2, details: 'Code Academy: HTML & CSS Tutorial'},
+	{step: 3, details: 'Code Academy: Responsive Design Tutorial'},
+	{step: 4, details: 'Create a Web Page'},
+	{step: 5, details: 'Iterate a Web Page'}
+]
```
Comment (LouiaR) [inline frontend/src/stepsArray.js]: Great :+1: Job @OlenaKashuba  
what do you think to turn this files in JSON instead?

Comment (OlenaKashuba) [reply frontend/src/stepsArray.js]: not sure why, any ideas?

Comment (LouiaR) [reply frontend/src/stepsArray.js]: It thinks it's just an array of data.

Comment (OlenaKashuba) [reply frontend/src/stepsArray.js]: yes, a variable you can use for your component too. If there are any changes in it, we will change it in one place

**Comment by dahfool** [comment]
@OlenaKashuba @LouiaR Its an array of objects.

**Pull request merged**